packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -619,42 +619,10 @@ contract VotingV2 is
         commitAndEmitEncryptedVote(identifier, time, "", hash, encryptedVote);
     }
 
-    /**
```

Comment:
I moved this function to Staker because all the state delegateToStaker and voterStakes is in the latter contract.